release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -816,6 +899,16 @@ In the table, features are marked with the following statuses:
 |REM
 |REM
 
+|Grafana component in monitoring stack
+|
+|DEP
+|REM
+
+|Access to Prometheus and Grafana UIs in monitoring stack
```

Comment:
for the removed UIs, in 4.10 doc, we mentioned the removed alertmanager UI directly like
https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-monitoring-alertmanager
for 4.11, the removed UIs are mentioned in `Deprecated and removed features tracker` table, it also make sense, but the former way is easier to find the 4.11 Prometheus and Grafana UIs are removed